designs/2023-test-rule-errors/README.md

```diff
@@ -214,33 +224,37 @@ This new feature is a non-breaking change and has no impact unless a user choose
     projects have already implemented a similar feature.
 -->
 
-### Top-level `error` array
+### Reuse `invalid` test cases
```

Comment:
I'm not sure I understand the use case for this. If we are saying that invalid test cases fail linting and fatal ones throw errors, it doesn't seem like there is an overlap here and adding a second way to test the same thing seems more likely to lead to confusion.

Is there some reason we need this capability?

Reply:
This is under the "Alternatives" sections, not intended to the proposed design. I added a note in this alternative explaining that we rejected it during discussion:

> During the RFC discussion, we decided against this alternative because fatal test cases are sufficiently-different from invalid test cases that they deserve to have their own separate array to avoid confusion.

Reply:
Ah sorry! Only viewing the diff from last time confused me a bit.